play-json/src/main/scala/play/api/libs/json/Reads.scala

```diff
@@ -604,6 +604,18 @@ trait DefaultReads extends LowPriorityDefaultReads {
     instantReads(DateTimeFormatter.ISO_DATE_TIME)
 
   /**
+   * Reads for the `java.time.ZoneId` type.
+   */
+  implicit val ZoneIdReads: Reads[ZoneId] = Reads[ZoneId] {
+    case JsString(s) => try {
+        JsSuccess(ZoneId.of(s))
+      } catch {
+        case _: DateTimeException => JsError(JsonValidationError("error.expected.timezone", s))
+      }
+    case _ => JsError(JsonValidationError("error.expected.jsstring"))
```

Comment:
Are you asking me to put a blank like before `case _ => JsError...` ? I can't find an example in the rest of the code. 

Would this be the right formatting ?
```scala
 implicit val ZoneIdReads: Reads[ZoneId] = Reads[ZoneId] {
    case JsString(s) => try {
      JsSuccess(ZoneId.of(s))
    } catch {
      case _: DateTimeException => JsError(JsonValidationError("error.expected.timezone", s))
    }
      
    case _ => JsError(JsonValidationError("error.expected.jsstring"))
  }
```